autobotAI_integrations/deep_agent_schema.py

```
@@ -385,3 +456,11 @@ class DeepAgentResponse(BaseModel):
             "from the agent interrupt: {action_requests: [...], review_configs: [...]}"
         ),
     )
+    goal_complete: bool = Field(
+        False,
```

Ruff (0.15.17)

[warning] Boolean positional value in function call

(FBT003)